### ISSUE TITLE
Research Mask Detection Models

### ISSUE THREAD

**Issue by waqarws** [waqarws/EfficientDetonVideo#1]
Mask Detection Using Image Processing
- Can be a binary CNN classifier
- Is Pre-trained model available?
- Is training data available/ can it be created?
